kong/plugins/http-log/handler.lua

```diff
@@ -189,12 +193,43 @@ function HttpLogHandler:log(conf)
 
   local queue_conf = Queue.get_plugin_params("http-log", conf, make_queue_name(conf))
   kong.log.debug("Queue name automatically configured based on configuration parameters to: ", queue_conf.name)
+  local entry = cjson.encode(kong.log.serialize())
+
+  if queue_conf.max_batch_size == 1 then
+    local ok ,err = timer_at(0, function()
```

Comment:
If the TPS is very high, the number of timers would soar.

Reply:
I suppose all the timers would be managed by `timer-ng` since 3.0. In that case, there will be warning logs from the `timer-ng`.